boards/laird_connect/bl654_dvk/bl654_dvk_nrf52840_pa_defconfig

```diff
@@ -0,0 +1,26 @@
+# SPDX-License-Identifier: Apache-2.0
+
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# Enable RTT
+CONFIG_USE_SEGGER_RTT=y
+
+# enable GPIO
+CONFIG_GPIO=y
+
+# enable uart driver
+CONFIG_SERIAL=y
+
+# enable console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
```

Comment:
These are not needed, the base `bl654_dvk_defconfig` file will be used and these will be appended, so only things needed here are things the base file does not have, if there's nothing then the file can be deleted